source/system-administrators/activities/security/security-change-defaults.rst

```diff
@@ -18,11 +18,23 @@ Crafter CMS installations are pre-configured with default passwords, tokens, key
         :caption: *CRAFTER_HOME/bin/crafter-setenv.sh*
 
         # -------------------- Encryption variables --------------------
+        # These variables are used to encrypt and decrypt values inside the configuration files.
         export CRAFTER_ENCRYPTION_KEY=${CRAFTER_ENCRYPTION_KEY:="default_encryption_key"}
         export CRAFTER_ENCRYPTION_SALT=${CRAFTER_ENCRYPTION_SALT:="default_encryption_salt"}
 
      |
 
+* Replace the default values for database values encryption key and salt before starting Crafter CMS for the very first time.  Remember that these values should **not** be changed after Crafter CMS has been started.
```

Comment:
I think we can say they can be changed. They can't be changed if one of the following occurs: remote repo passwords and keys, and cluster passwords and keys.